inc/thorvg.h

```diff
@@ -1821,6 +1821,23 @@ class TVG_API Saver final
      */
     Result save(std::unique_ptr<Paint> paint, const std::string& path, bool compress = true) noexcept;
 
+    /**
+     * @brief Exports the given @p animation data to the given @p path
+     *
+     * @param[in] animation The animation to be saved with all its associated properties.
+     * @param[in] path A path to the file, in which the animation is to be saved.
+     * @param[in] frameDelay The interval between each frame(ms). Default value is 2(ms).
+     *
+     * @retval Result::Success When succeed.
+     * @retval Result::InsufficientCondition If currently saving other resources.
+     * @retval Result::NonSupport When trying to save a file with an unknown extension or in an unsupported format.
+     * @retval Result::MemoryCorruption An internal error.
+     * @retval Result::Unknown In case an empty paint is to be saved.
+     *
+     * @BETA_API
+     */
+    Result save(std::unique_ptr<Animation> animation, const std::string& path, int frameDelay = 2) noexcept;
```

Comment:
The inclusion of 'frameDelay' as a parameter seems unnecessary since 'Animation' already contains frame-related information, such as the number of frames and their duration. 'frameDelay' becomes relevant only when dealing with multiple raw scene frames that lack animation timing information. Not here.

Reply:
You're right. The delay of gif.h is not ms, it is 1/100th of a second. I will modify it.